lib/subscription_payment_api/providers/braintree/transaction.rb

```
@@ -21,6 +21,18 @@ def refund(id:)
           raise SubscriptionPaymentApi::Exceptions::GeneralError.new(e.message)
         end
 
+        sig do
+          params(id: String)
+            .returns(SubscriptionPaymentApi::Entity::Transaction)
+        end
+        def find(id:)
+          response = gateway.transaction.find(id)
+
+          to_transaction(response)
+        rescue => e
```

rubocop (1.68.0)

[convention] Avoid rescuing without specifying an error class.

(Style/RescueStandardError)